src/main/scala/rps/model/Play.scala

```diff
@@ -1,3 +1,6 @@
 package rps.model
 
-case class Play(userMove: Move, computerMove: Move, result: Result)
+import java.util.UUID
+import java.time.Instant
+
+case class Play(id: UUID, userMove: Move, computerMove: Move, result: Result, createdAt: Instant)
```

Comment:
Not sure `id` and `createdAt` needs to be included in this model. We're not using them in the client, and you're forced to create a random Id and an `Instant.now` in the service, that it's an anti-pattern IMHO.
In real apps, you would have two different data classes for reads and writes (see the `Entity` wrapper in GDSM).
In this case, I'd prefer to avoid this complexity and keep this class as it was before

Reply:
Hum... in PlayRow model that fields are useful. I need to create a random UUID and an Instant.now before inserting the PlayRow into the database. So where is a good place to put this logic? Maybe in the function that convert a Play to a PlayRow?

Reply:
yep, that's a good point.